src/components/ProfilePage/useAddressMenuContent.tsx

```diff
@@ -0,0 +1,72 @@
+import { useMenuStore } from '@/stores/menu';
+import { useAccount } from '@lifi/wallet-management';
+import OpenInNewIcon from '@mui/icons-material/OpenInNew';
+import ReceiptLongIcon from '@mui/icons-material/ReceiptLong';
+import { useTheme } from '@mui/material/styles';
+import { useRouter } from 'next/navigation';
+import { useTranslation } from 'react-i18next';
+import { TrackingAction, TrackingCategory } from 'src/const/trackingKeys';
+import { JUMPER_SCAN_PATH } from 'src/const/urls';
+import { useUserTracking } from 'src/hooks/userTracking';
+
+export const useAddressMenuContent = () => {
+  const { account } = useAccount();
+  const { trackEvent } = useUserTracking();
+  const { t } = useTranslation();
+  const theme = useTheme();
+  const isDarkMode = theme.palette.mode === 'dark';
+  const closeAllMenus = useMenuStore((state) => state.closeAllMenus);
+  const router = useRouter();
+
+  const handleScanButton = () => {
+    const url = `${JUMPER_SCAN_PATH}/wallet/${account.address}`;
+    trackEvent({
+      category: TrackingCategory.AddressMenu,
+      action: TrackingAction.OpenJumperScan,
+      label: 'open-jumper-scan-wallet',
+    });
+    router.push(url);
+  };
+
+  return [
+    {
+      label: t('profile_page.open', { tool: 'explorer' }) as string,
+      showMoreIcon: false,
+      prefixIcon: (
+        <OpenInNewIcon
+          sx={{
+            height: '16px',
+            color: isDarkMode
+              ? theme.palette.white.main
+              : theme.palette.black.main,
+          }}
+        />
+      ),
+      onClick: () => {
+        closeAllMenus();
+      },
+      link: {
+        url: `https://etherscan.io/address/${account.address}`,
+        external: true,
+      },
+    },
+    {
+      label: t('profile_page.open', { tool: 'Jumper Scan' }) as string,
+      showMoreIcon: false,
+      prefixIcon: (
+        <ReceiptLongIcon
+          sx={{
+            height: '16px',
+            color: isDarkMode
+              ? theme.palette.white.main
+              : theme.palette.black.main,
+          }}
+        />
+      ),
+      onClick: () => {
+        handleScanButton();
```

Comment:
Would be good to have a link as well (as above)

Reply:
This is only available once a user connects his wallet + when the menu is opened. Is not part of the DOM otherwise. We may come back and refactor if necessary, in a different ticket.